hardware/Comply-to-the-Regulations.md

```diff
@@ -0,0 +1,394 @@
+  - [3.2 CE Regulation](#32-CE-Regulation)
+- [4. Basic RF test items and limits for complying the regulations](#4-Basic-RF-test-items-and-limits-for-complying-the-regulations)
+  - [4.1 FCC requirements in Sub-GHz bands](#41-FCC-requirements-in-Sub-GHz-bands)
+  - [4.2 FCC requirements in 2.4GHz band](#42-FCC-requirements-in-2.4GHz-band)
+  - [4.3 CE requirements in Sub-GHz bands](#43-CE-requirements-in-Sub-GHz-bands)
+  - [4.4 CE requirements in 2.4GHz band](#44-CE-requirements-in-2.4GHz-band)
+- [5. Customer regulatory cerfitication flow](#5-Customer-regulatory-cerfitication-flow)
+  - [5.1 Silicon Labs provides documents for certification](#51-Silicon-Labs-provides-documents-for-certification)
+  - [5.2 Process flow in certifications](#52-Process-flow-in-certifications)
+  - [5.3 Engineering flow](#53-Engineering-flow)
+  - [5.4 Paperwork flow](#54-Paperwork-flow)
+
+</details>
+
+
+# 1. Introduction
+After the customer finishes the product design, it should consider how to test the product and let it comply to the RF regulations. RF conducted or Radiated measurements should pass the regulation limits or requirements for product intended to be sold in regions. It is called complying to the regulations or certifications. The regulations depend on different regions or countries. There are chips/modules, Sub-GHz/2.4GHz solutions in Silicon Labs' production catalog, and there are many protocols related product such as proprietary,Zigbee/thread,BLE, and Z-Wave should be classified. Thus the detail regulatory certifications maybe different from application to application and from country to country. 
+
+# 2. Typical regulatory certification regions
+Silicon Labs has many products such as chips, SiP modules, and PCB modules. And the products support main sub-GHz bands(169MHz,434MHz,868MHz and 915MHz) and 2.4GHz frequenc band. These different products and frequency bands almost comply to regulations from any region in the world. There are many region regulations such as [CE](http://ec.europa.eu/growth/sectors/electrical-engineering/red-directive_fi) (Europe), [FCC](https://www.fcc.gov/)(USA), [ISED](https://www.canada.ca/en/innovation-science-economic-development.html)(Canada), [MIC](http://www.soumu.go.jp/english/)(Japan), [KC](http://eng.kcc.go.kr/user/ehpMain.do)(South Korea) and other regions. 
+
+And there are some test houses provide certification service for the regions as following:
+<div align="center">
+  <img src="files\HW-Comply-to-the-Regulations\Test-House.png">  
+</div> 
+
+For the certification activeties, you can directly contact the region authorities:
+<div align="center">
+  <img src="files\HW-Comply-to-the-Regulations\Authorities-Contacts.png">  
+</div>
+
+In the following introduction, it will mainly focus on EC and FCC regulations. For Other region regulatory certifications, please check [AN1048](https://www.silabs.com/documents/public/application-notes/an1048-regulatory-certifications.pdf) for more details.
```

Comment:
Change "EC" to "CE".  Change "Other" to "other".

Reply:
Corrected.